codecov.yml

```diff
@@ -0,0 +1,11 @@
+coverage:
+  status:
+    project:
+      default:
+        informational: true
+    patch:
+      default:
+        informational: true
+
+# When modifying this file, please validate using
+# curl -X POST --data-binary @codecov.yml https://codecov.io/validate
```

Comment:
Thanks for taking the lead on this one :heart:!

Can we add some [ignore paths](https://docs.codecov.com/docs/ignoring-paths)? E.g. `**/zz_generated.deepcopy.go`

Do you know if there's a way to make codecov to NOT add annotations to the file? If you scroll down in the [PR/files view](https://github.com/external-secrets/external-secrets/pull/2995/files).
It's also visible in the [GH check run](https://github.com/external-secrets/external-secrets/pull/2995/checks?check_run_id=20123445906).
The problem is, that this makes reviewing really tedious to scroll over the annotations :/ I'd prefer to NOT add any annotations. Found the [docs related to this here](https://docs.codecov.com/docs/github-checks#yaml-configuration-for-github-checks-and-codecov).

Reply:
Thanks for your feedback, @moolen!
I've turned off the annotations and configured ignore paths. Would you take a look at it again when you have time?